test/node-unit/cli/config.spec.js

```diff
@@ -57,6 +57,17 @@ describe('cli/config', function() {
         });
       });
 
+      describe('when supplied a filepath with .jsonc extension', function() {
```

Comment:
It would be nice if all suite titles double-quoted the file extension
(e.g., `when supplied a filepath with ".jsonc" extension`)